__tests__/database/migration/index.test.ts

```diff
@@ -5,7 +5,7 @@ describe('TESTING migration file', () => {
     expect(DROP_ALL_DB).toBeFalsy();
   });
 
-  it('TEST should check TARGET_MIGRATION_VERSION to 5', () => {
-    expect(TARGET_MIGRATION_VERSION).toBe(6);
+  it('TEST should check TARGET_MIGRATION_VERSION to 7', () => {
+    expect(TARGET_MIGRATION_VERSION).toBe(7);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The test case description and the assertion do not match. The description says it should check `TARGET_MIGRATION_VERSION` to 7, but the assertion is checking if `TARGET_MIGRATION_VERSION` is equal to 7. It would be more accurate to say "should expect `TARGET_MIGRATION_VERSION` to be 7".
```diff
-   it('TEST should check TARGET_MIGRATION_VERSION to 7', () => {
+   it('TEST should expect TARGET_MIGRATION_VERSION to be 7', () => {
      expect(TARGET_MIGRATION_VERSION).toBe(7);
    });
  });
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->